common/Server/eos/core/eos_core.sqf

```diff
@@ -107,14 +119,21 @@ sleep 0.25;
 	for "_counter" from 1 to _dGrps do {
 	if (isnil "_dGrp") then {_dGrp=[];};
 
-		_newpos=[_mkr,50] call EOS_fnc_findSafePos;
+		_newpos=[_mkr,100] call EOS_fnc_findSafePos;
 			if (surfaceiswater _newpos) then {_vehType=8;}else{_vehType=2;};
 
 					_dGroup=[_newpos,_side,_faction,_vehType]call EOS_fnc_spawnvehicle;
 					0=[(_dGroup select 0),[4, 10],(_dGroup select 2),_faction,9] call eos_fnc_setcargo;
 
 						0=[(_dGroup select 2),"ARMskill"] call eos_fnc_grouphandlers;
-						0 = [(_dGroup select 2),_mkr] call EOS_fnc_taskpatrol;
+						
+						
+						if(isClass(configfile >> "lambs_main_Display")) then {
```

Comment:
We should probably do the same format as detecting ace though the mod path config just encase lambs changes